### PR TITLE
Fix several typos in wml

### PR DESCRIPTION
**Pull request by hexagonrecursion**

### CODE REVIEW

File: data/core/encyclopedia/calendar.cfg [hunk 1]
```diff
@@ -22,7 +23,7 @@ Summit Star
 Kindlefire
 Stillseed
 Reaper’s Moon
-Verglas Bloom
+Vergla's Bloom
```
Comment (stevecotton) [inline data/core/encyclopedia/calendar.cfg:26]: Needs a typographic-style quotation mark here.

@nemaara is this intended to be a possessive, or is "Verglas" meant to be a word?

File: data/core/units/undead/Skele_Bone_Shooter.cfg [hunk 2]
```diff
@@ -13,7 +13,7 @@
     alignment=chaotic
     advances_to=Banebow
     cost=26
-    description= _ "Of a dark sorcerer’s creations, some take more strongly to the false life given them. The potency of their un-life is given equipment to match; archers, in particular, are often outfitted with a truly vile arsenal. Their quivers are filled with shafts made not of wood, but of the bones of their victims. It follows that they are dubbed simply ‘Bone-Shooters’ by their unfortunate enemies."
+    description= _ "Of a dark sorcerer’s creations, some take more strongly to the false life given them. The potency of their unlife is given equipment to match; archers, in particular, are often outfitted with a truly vile arsenal. Their quivers are filled with shafts made not of wood, but of the bones of their victims. It follows that they are dubbed simply ‘Bone-Shooters’ by their unfortunate enemies."
```
Comment (stevecotton) [inline data/core/units/undead/Skele_Bone_Shooter.cfg:16]: I checked for other instances of "unlife" and "un-life", and Wesnoth canon seems to be equally split between the two spellings (as in, there are more corrections needed whichever one we decide to use).

@nemaara which of the two spellings should be our house-style?

**Review by stevecotton — COMMENTED** [review]
Thanks for catching these.

Although I'm reviewing, I'd be the wrong person to approve this - likely that should be @nemaara .

File: data/core/encyclopedia/calendar.cfg (hunk 1 above)
Comment (hexagonrecursion) [reply data/core/encyclopedia/calendar.cfg]: > Needs a typographic-style quotation mark here.

Done

Comment (nemaara) [reply data/core/encyclopedia/calendar.cfg]: Verglas is a word. Keep it as written.

File: data/core/units/undead/Skele_Bone_Shooter.cfg (hunk 2 above)
Comment (nemaara) [reply data/core/units/undead/Skele_Bone_Shooter.cfg:16]: I prefer unlife, it reads cleaner. Extra punctuation in unnecessary places is hard to read for me personally.

File: data/core/encyclopedia/calendar.cfg (hunk 1 above)
Comment (stevecotton) [reply data/core/encyclopedia/calendar.cfg]: Okay, I should have checked a dictionary before writing my previous comment. However, words that are far less obscure than this one are typically given translation hints.

Comment (hexagonrecursion) [reply data/core/encyclopedia/calendar.cfg]: done

**Review by nemaara — APPROVED** [review]

**Pull request merged**